components/brave_ads/core/internal/geographical/subdivision/subdivision_url_request.cc

```diff
@@ -24,11 +25,14 @@ namespace brave_ads {
 
 namespace {
 
-constexpr base::TimeDelta kFetchAfter = base::Days(1);
 constexpr base::TimeDelta kDebugFetchAfter = base::Minutes(5);
 
 constexpr base::TimeDelta kRetryAfter = base::Minutes(1);
 
+base::TimeDelta GetFetchInterval() {
```

Comment:
nit: maybe remove and just call `kSubdivisionFetchIntervalMinutes.Get()` where it is used